src/clothing_mod.h

```diff
@@ -6,6 +6,7 @@
 #include <string>
 #include <vector>
 #include <array>
+#include <set>
```

Comment:
Why are you including set?  You don't appear to use it.

Reply:
Good catch, I was using set for apply_flags and exclude_flags, but switched to vector later for easier interplay with item.cpp. I'll remove this in next commit.